sdk/nodejs/resource.ts

```diff
@@ -327,12 +327,21 @@ export class Output<T> {
     /* @internal */ public readonly promise: () => Promise<T>;
 
     /**
-     * The list of resource that this output value depends on.
+     * The list of statically-known resources that this output value depends on. Notably, this does
+     * *not* include dependencies lifted from the result of an apply callback.
```

Comment:
Should we add a comment in here about how this is for back-compat, so that it's clear why we structured this way for future readers?

Reply:
Will do.